### ISSUE TITLE
Glutin example? 

### ISSUE THREAD

**Issue by Hperigo** [Gekkio/imgui-rs#173]
Hi, is there an example of how to use this library with glutin only?  

thanks

**Comment by malikolivier** [comment]
What do you exactly want to do?
You need a rendering back-end (e.g. glium or gfx). glutin takes care of managing GUI events.

All the examples here use glium (and glutin, as a dependency of glium):
https://github.com/Gekkio/imgui-rs/tree/master/imgui-glium-examples

Is there anything that those examples do not provide that you want?

**Comment by Hperigo** [comment]
Hi, thanks for the reply.
I was doing some experiments (learning) with rust and messing around with opengl & glutin.

I did tried glium but it takes a rather long time to compile and I was wondering if was possible to use just glutin and raw gl calls, that's all imgui needs to render if I'm not mistaken. 

I did tried this crate:[rust-imgui-opengl-renderer]( https://github.com/michaelfairley/rust-imgui-opengl-renderer).  but it only worked with the SDL "backend". Because I'm only messing around I ended up going with the sdl instead of glutin. 

anyway, it would  be nice to have a render option like without extra dependencies.
